Agents/PropertyValueEstimationAgent/README.md

```diff
@@ -0,0 +1,299 @@
+REGISTER_AGENT                # Boolean flag whether to register agent in KG (`true` required to detect derivations)
+```
+
+
+### **2) Accessing Github's Container registry**
+
+While building the Docker image of the agent, it also gets pushed to the [Container registry on Github]. **Please note** that this only applies to the stack-deployable images of the agent (i.e. images mentioned in section 1.3) and not the test images. Access needs to be ensured beforehand via your github [personal access token], which must have a `scope` that [allows you to publish and install packages]. To log in to the [Container registry on Github] simply run the following command to establish the connection and provide the access token when prompted:
+```
+  docker login ghcr.io -u <github_username>
+  <github_personal_access_token>
+```
+
+### **3) VS Code specifics**
+
+In order to avoid potential launching issues using the provided `launch.json` and `tasks.json` shell commands, please ensure the `augustocdias.tasks-shell-input` plugin is installed.
+
+
+&nbsp;
+## 1.2 Spinning up the core stack
+
+Navigate to `Deploy/stacks/dynamic/stack-manager` and run the following command there from a `bash` terminal. To [spin up the stack], both a `postgis_password` and `geoserver_password` file need to be created in the `stack-manager/inputs/secrets/` directory (see detailed guidance following the provided link). There are several [common stack scripts] provided to manage the stack:
+
+```bash
+# Start the stack (please note that this might take some time)
+bash ./stack.sh start <STACK NAME>
+
+# Stop the stack
+bash ./stack.sh stop <STACK NAME>
+
+# Remove stack services (incl. volumes)
+bash ./stack.sh remove <STACK_NAME> -v
+```
+
+After spinning up the stack, the GUI endpoints to the running containers can be accessed via Browser (i.e. adminer, blazegraph, ontop, geoserver). The endpoints and required log-in settings can be found in the [spin up the stack] README.
+
+
+&nbsp;
+## 1.3 Deploying the agent to the stack
+
+This agent requires the [JPS_BASE_LIB] and [Stack-Clients] to be wrapped by [py4jps]. Therefore, after installation of all required packages (incl. `py4jps>=1.0.29`), the `StackClients` resource needs to be added to allow for access through `py4jps`. All required steps are detailed in the [py4jps] documentation. However, the command below below shall suffice to compile the latest `StackClients` resource locally to be available for installation into the Docker container using the provided [Dockerfile]. Please note, that compiling requires a [Java Development Kit version >=11].
+```bash
+# Compiling latest Stack_Clients resource for py4jps
+bash ./build_py4jps_stackclients_resource.sh
+```
+
+Simply execute the following command in the same folder as this `README` to build and spin up the *production version* of the agent (from a `bash` terminal). The stack `<STACK NAME>` is the name of an already running stack.
```

Comment:
Can you add a note about making the visibility of the pushed docker images public?

Reply:
Added in commit [606def7](https://github.com/cambridge-cares/TheWorldAvatar/pull/454/commits/606def74ec3b921e3c451767b78458db3a2fa3d5)